modules/after-the-deadline.php

```diff
@@ -292,6 +292,13 @@ function AtD_should_load_on_page() {
 		return true;
 	}
 
+	/**
+	 * Allows scripts to be loaded via AtD in admin.
```

Comment:
Suggestion: Add a description that includes a bit more details. Something like "By default, AtD only enqueues JS on certain admin pages to reduce bloat. The filter allows additional pages to have AtD JS."